src/devices/video/psx.cpp

```diff
@@ -327,7 +327,7 @@ int psxgpu_device::DebugMeshDisplay( bitmap_rgb32 &bitmap, const rectangle &clip
 	if( m_debug.b_mesh )
 	{
 		for (int y = cliprect.min_y; y <= cliprect.max_y; y++)
-			draw_scanline16( bitmap, cliprect.min_x, y, cliprect.max_x+1, m_debug.mesh.pix16(y), pens() );
+			draw_scanline16( bitmap, cliprect.min_x, y, cliprect.max_x+1, &m_debug.mesh.pix16(y), pens() );
```

Comment:
Is the 4th parameter here (`length`) supposed to be a length in pixels, or maximum coordinate?  If it's a length in pixels, you'll need to subtract `cliprect.min_x` to make it correct.